src/oscar/templates/oscar/customer/emails/commtype_order_placed_body.html

```diff
@@ -1,28 +1,65 @@
+        <p>{% blocktrans with order_number=order.number %}We are pleased to confirm your order {{ order_number }} has been received and
+        will be processed shortly.{% endblocktrans %}</p>
+    </td>
+</tr>
 
-<p>{% blocktrans with order_number=order.number %}We are pleased to confirm your order {{ order_number }} has been received and
-will be processed shortly.{% endblocktrans %}</p>
+<tr>
+    <td class="content-block">
+        <table class="order">
+            <tbody><tr>
+                <td>{% trans 'Your order contains:' %}</td>
+            </tr>
+            <tr>
+                <td>
+                    <table class="order-items" cellpadding="0" cellspacing="0">
+                        <tbody>
+                            {% for line in order.lines.all %}
+                                <tr>
+                                    <td>{{ line.title }} &times; {{ line.quantity }}</td>
+                                    <td class="alignright">{{ line.line_price_incl_tax|currency:order.currency }}</td>
+                                </tr>
+                            {% endfor %}
+                            <tr>
+                                <td class="alignright">{% trans 'Basket total:' %}</td>
+                                <td class="alignright">{{ order.basket_total_incl_tax|currency:order.currency }}</td>
+                            </tr>
+                            <tr>
+                                <td class="alignright">{% trans 'Shipping:' %}</td>
+                                <td class="alignright">{{ order.shipping_incl_tax|currency:order.currency }}</td>
+                            </tr>
+                            <tr class="total">
+                                <td class="alignright" width="80%">{% trans 'Order Total:' %}</td>
+                                <td class="alignright">{{ order.total_incl_tax|currency:order.currency }}</td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </td>
+            </tr>
+        </tbody></table>
+    </td>
+</tr>
 
-<p>{% trans 'Your order contains:' %}</p>
+<tr>
+    <td class="content-block">
+        <p>{% trans 'Shipping address:' %}</p>
+        <p>{% for field in order.shipping_address.active_address_fields %}  {{ field }}<br/>
+        {% endfor %}</p>
+    </td>
+</tr>
 
-<ul>
-    {% for line in order.lines.all %}
-    <li>{{ line.title }} - {%  trans 'quantity:' %} {{ line.quantity }} - {% trans 'price:' %} {{ line.line_price_incl_tax|currency:order.currency }}</li>
-    {% endfor %}
-</ul>
-<p>
-{% trans 'Basket total:' %} {{ order.basket_total_incl_tax|currency:order.currency }}<br/>
-{% trans 'Shipping:' %} {{ order.shipping_incl_tax|currency:order.currency }}<br/>
-{% trans 'Order Total:' %} {{ order.total_incl_tax|currency:order.currency }}
-</p>
+<tr>
+    <td class="content-block">
+        {% if status_url %}
```

Comment:
The if statement should wrap the outer `tr`, otherwise you will display an empty row.

Reply:
Indeed.